tasks/bootstrap_local_vm/05_add_host.yml

```diff
@@ -126,16 +126,47 @@
     retries: 120
     delay: 5
   - debug: var=host_result_up_check
-  - name: Check host status
-    fail:
-      msg: >
-        The host has been set in non_operational status,
-        please check engine logs,
-        fix accordingly and re-deploy.
+  - name: handle non_operational failure
+    block:
+    - set_fact: host_id={{ host_result_up_check.ansible_facts.ovirt_hosts[0].id }}
+    - name: Return all events of type greater then error
+      ovirt_event_facts:
+        auth: "{{ ovirt_auth }}"
+        search: "severity>=error"
+      register: error_events
+
+    - name: extract the description from the events using host.id and unique code
```

Comment:
same (use a capital case on the first letter)

Reply:
done